### ISSUE TITLE
0.9.29: trakt accesses fail always with HTTP error()

### ISSUE THREAD

**Issue by endlessterror** [scakemyer/plugin.video.quasar#387]
Just discovered that the context menu added options to integrate with trakt. Awesome work.

Problem is that I did do the authorize dance but received an HTTP Error. I entered the quasar options, set up my trakt account, did the authorization again. No matter what I do, I am always asked to authorize, even though I have already done that. Each time, Quasar responds with HTTP error(). Could it be that I am not having an English locale?

I do have 0.9.29 installed. Attaching kodi.log
[kodi.zip](https://github.com/scakemyer/plugin.video.quasar/files/170987/kodi.zip)

**Comment by ShlomiD83** [comment]
got that error with english to, so i dont think its a language thing. tried a couple of times until got "woohoo" from quasar.

**Comment by hfmls** [comment]
same error, with portuguese, after woohoo i got http error too.

**Comment by tomtomclub** [comment]
I reboot end remover all cachet works fine now

**Comment by tomtomclub** [comment]
The best addon four renove alle cache  
2.0.2  dutch crew

**Comment by endlessterror** [comment]
I rebooted, entered Quasar, removed temp files. Then I used another addon (maintenance tool from tvaddons) to clean cache files. I tried to access my Trakt watchlist and was asked to reactivate. I did so, but this time the watchlist list opened up just fine!

Seems like @tomtomclub 's suggestion works just fine!

Perhaps I should modify #366 to add a reference to clean cache files upon certain, troublesome updates.

**Comment by seltun** [comment]
This is also happening to me but I don't get an Woohoo notification.
Fresh Kodi install, installed quasar v0.9.29 and restarted Kodi.

First tried to access the collection option. Trakt authorize pop up came up, I used a browser to access the address, inserted the key, it redirected to a web page with an Woohoo message, then I clicked OK on the Trakt authorize popup but got a notification saying Navigation Error: HTTP error(). Tried to access again the collection option but the popup showed up again.

Then I tried with the authorize option in the settings but got the same error and the collection still showed the pop up.

[kodi.log](http://xbmclogs.com/pvxetjb4c)

**Comment by scakemyer** [comment]
Sorry for the mess guys, I would've loved to keep it simple, but I had to go ahead and create CloudHole so I could re-use my CloudFlare cookies within the add-on, just to be able to test Trakt authorization and list integration. It ended up a bit more complicated than I hoped, and caused issues to those not affected by that problem, but should be mostly fixed in 0.9.30 and those like me that also get constantly blocked by CloudFlare will be able to use it too.

**Comment by endlessterror** [comment]
Just updated an openelec 6.0-based raspberry to version 0.9.30. For the record, I did not try to use any trakt auth on 0.9.29, nor I tried to clean cache etc, in order to be able to check whether the next version (0.9.30) worked immediately. 

What happened is that after the woohoo from trakt, a woohoo from quasar came up momentarily, followed by a HTTP-Error () warning. But after that, everything worked just fine.

So, those of us experiencing this issue will most likely need to do nothing else, apart from installing 0.9.30.
